src/components/Plants/SpeciesByPlotChart.tsx

```diff
@@ -26,12 +29,17 @@ export default function SpeciesByPlotChart({ plots }: Props): JSX.Element {
   const onChangePlot = (newValue: string) => {
     if (plots) {
       setSelectedPlot(plots.find((p) => p.fullName === newValue));
+      updatePlotPreferences(newValue);
```

Comment:
use the plot id
```
if (plots) {
  const plot = plots.find(p => p.fullName === newValue);
  setSeletedPlot(plot);
  updatePlotPreferences(plot.id)
}
```
also, can we use the id instead of fullname for the onChangePlot value?
We can use the fullname for the label